scripts/benchmarks_ci.py

```diff
@@ -142,6 +143,12 @@ def __is_valid_datetime(dt: str) -> str:
             "%%Y-%%m-%%dT%%H:%%M:%%SZ").'''
     )
 
+    parser.add_argument('--upload-to-perflab-container',
```

Comment:
One last thing you could do when using `argparse` is to add validation to the user input. Currently, the argument does not explain that the input is a "path" (I would rename the argument), and maybe validate that the path exist?

Reply:
It turns out that is not a path on disk but is a path in the results blob. It's also sort of optional - if you just specify `--upload-to-perflab-contanier` with no argument it will go to the root of the results container. (That may never be what we want but it might be?) Gonna leave it.